velox/experimental/cudf/exec/DebugUtil.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/experimental/cudf/vector/CudfVector.h"
+
+namespace facebook::velox::cudf_velox {
+class DebugUtil {
+ public:
+  std::string
+  toString(const cudf::table_view& table, vector_size_t from, vector_size_t to);
```

Comment:
Is there a strong reason to make this a class rather than just a function? Is it so you can hide the creation of the memory pools and then re-use them in multiple `toString` calls?

Reply:
Yes, the class DebugUtil holds the memory pool. And usually in Velox, we use class to construct the functions such as `DecimalUtil`.